ui/frontend/BuildMenu.tsx

```diff
@@ -55,7 +57,11 @@ const BuildMenu: React.SFC<BuildMenuProps> = props => {
         Build and show the resulting LLVM IR, LLVM’s intermediate representation.
       </ButtonMenuItem>
       <ButtonMenuItem name="MIR" onClick={compileToMir}>
-        Build and show the resulting MIR, Rust’s intermediate representation.
+        Build and show the resulting MIR, Rust’s intermediate representation for control flow.
```

Comment:
```suggestion
        Build and show the resulting MIR, Rust’s control-flow-based intermediate representation.
```

Reply:
I don't think this is a good description because it implies MIR can represent a whole crate. MIR can *only* represent function bodies or const initializers.

Reply:
How does my change imply any more than how it was before that MIR can represent a whole crate?

Reply:
Yeah I don't know what I was thinking lol, this seems fine too.